tests/python_tests/common.py

```diff
@@ -12,11 +12,12 @@
 from transformers import GenerationConfig as HFGenerationConfig
 from typing import List, Tuple
 
+TESTS_ROOT = Path(__file__).parent
 
 def get_greedy() -> GenerationConfig:
     generation_config = GenerationConfig()
     generation_config.num_return_sequences = 1
-    generation_config.max_new_tokens = 30
+    generation_config.max_new_tokens = 300
```

Comment:
should we keep as is to minimize amount of time we spend on precommit tests?

Reply:
Thanks for spotting this, these are all leftovers from my initial testing